ccd-definition/CaseField/CaseField-SDO-R2-nonprod.json

```diff
@@ -88,5 +88,54 @@
     "FieldType": "JudgeDecisionOnReconsiderationSelect",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "smallClaimsFlightDelayTitle",
+    "Label": "## Flight Delay ",
+    "FieldType": "Label",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "smallClaimsFlightDelayToggle",
+    "Label": " ",
+    "FieldType": "MultiSelectList",
+    "FieldTypeParameter": "OrderDetailsPagesSectionsToggle",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "smallClaimsFlightDelay",
+    "Label": " ",
+    "FieldType": "SmallClaimsFlightDelay",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "smallClaimsFlightDelayEndLine",
+    "Label": "****",
+    "FieldType": "Label",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "smallClaimsFlightDelayTittle1",
```

Comment:
Do we need to define it here as its already define as part of complex field types?